tutorials/setting-up-lxd-1604/setting-up-lxd-1604-tutorial.md

```diff
@@ -0,0 +1,211 @@
+---
+id: tutorial-setting-up-lxd-1604
+summary: Install and configure the LXD container hypervisor on Ubuntu 16.04
+categories: lxd
+tags: tutorial,installation,ubuntu,desktop,server,lxd
+difficulty: 2
+status: Published
+published: 2017-06-29
+
+---
+
+# Setting up LXD on Ubuntu 16.04
+
+## Overview
+Duration: 0:01
+
+This tutorial describes how to install and set up
+the [LXD container hypervisor](https://linuxcontainers.org/lxd) on Ubuntu 16.04
+(Xenial).
+
+![logo](images/containers.png)
+
+
+## Install LXD
+Duration: 0:04
+
+
+If you're running an Ubuntu 16.04 server installation, LXD is already
+preinstalled on your system.
+
+On Desktop installs, it needs to be installed manually. Open a terminal and run
+the following commands to install LXD:
+
+```bash
+$ sudo apt update
```

Comment:
We don't include the prompt, and separate commands from output - see https://github.com/canonical-websites/tutorials.ubuntu.com/blob/master/examples/guidelines-snap-tutorials.md#command-lines

Sorry if you have copied this from some existing tutorials - a few were bulk added and we need to retroactively make them fit the style